swan-lake/development-tutorials/source-code-dependencies/customizing-formatting.md

```diff
@@ -0,0 +1,389 @@
+---
+title: Customization of formatting
+description: The section on providing custom configurations to the formatter.
+keywords: ballerina, programming language, formatting, custom configurations
+permalink: /learn/customizing-formatting/
+active: customizing-formatting
```

Comment:
Can you update these also to be consistent with the rest of these pages, please?

Reply:
The `layout` section is missing here as per @sm1990, this is no longer required